src/sas/qtgui/Plotting/Slicers/SectorSlicer.py

```diff
@@ -273,6 +274,55 @@ def draw(self):
         """
         self.base.draw()
 
+    def getSlice(self, nbins=None):
+        """
+        """
+        data = self.data
+        # If we have no data, just return
+        if data is None:
+            return
+        # Averaging
+        from sas.sascalc.dataloader.manipulations import SectorQ
```

Comment:
This should now import from the `sasdata` package at `sasdata.data_util.manipulations`

Reply:
Made the changes: `from sas.sascalc.dataloader.manipulations import SectorQ` to `from sasdata.data_util.manipulations import SectorQ`